test/runtests.jl

```diff
@@ -135,7 +135,7 @@ end
         end
         i = findfirst(==(Base.RefValue{String}), src.stmts.type) # find allocation statement
         @assert !isnothing(i)
-        @test_broken is_return_escape(escapes.ssavalues[i])
+        @test is_return_escape(escapes.ssavalues[i])
```

Comment:
We may want to change this line 
https://github.com/aviatesk/EscapeAnalysis.jl/blob/f39d5ff601afcd4741b15e14db295330f8d268ff/src/EscapeAnalysis.jl#L422
into
```julia
        add_changes!(args[2:end], ir, AllEscape(), changes)
```
and keep these test cases remained to be broken until we address #15 ?